apollo-runtime/src/main/java/com/apollographql/apollo/cache/normalized/sql/RecordFieldAdapter.java

```diff
@@ -20,20 +20,24 @@
 import static com.apollographql.apollo.internal.json.ApolloJsonReader.bufferedSourceJsonReader;
 import static com.apollographql.apollo.internal.json.ApolloJsonReader.cacheJsonStreamReader;
 
-public final class FieldsAdapter {
+/**
+ * An adapter used to serialize and deserialize Record fields. Record object types will be serialized to
+ * {@link CacheReference}.
+ */
+public final class RecordFieldAdapter {
```

Comment:
Should we move this class out from `/sql` package as it's seems more generic and not specific to sql